frontend/pages/incomplete-uploads.vue

```diff
@@ -0,0 +1,174 @@
+            Metadata
+          </h2>
+          <div class="flex flex-column gap-2 w-fit">
+            <div class="flex gap-2 justify-content-between">
+              <span>Created At</span>
+              <b>{{ humanReadableTimeFromStandardString(slotProps.data.editorObject.value.createdAt.originalValue).value }}</b>
+            </div>
+            <div class="flex gap-2 justify-content-between">
+              <span>Ran At</span>
+              <b>{{ slotProps.data.editorObject.value.ranAt.originalValue ? humanReadableTimeFromStandardString(slotProps.data.editorObject.value.ranAt.originalValue).value : 'N/A' }}</b>
+            </div>
+            <div class="flex gap-2 justify-content-between">
+              <span>Completed At</span>
+              <b>{{ slotProps.data.editorObject.value.completedAt.originalValue ? humanReadableTimeFromStandardString(slotProps.data.editorObject.value.completedAt.originalValue).value : 'N/A' }}</b>
+            </div>
+            <div class="flex gap-2 justify-content-between">
+              <span>Failure Code</span>
+              <b>{{ slotProps.data.editorObject.value.failureCode.originalValue ?? 'N/A' }}</b>
+            </div>
+            <div class="flex gap-2 justify-content-between">
+              <span>Failure Message</span>
+              <b>{{ slotProps.data.editorObject.value.failureMessage.originalValue ?? 'N/A' }}</b>
+            </div>
+            <div class="flex gap-2 justify-content-between">
+              <span>ID</span>
+              <b>{{ slotProps.data.id }}</b>
+            </div>
+          </div>
+          <h2 class="mt-5">
+            Editable Properties
+          </h2>
+          <IncompleteuploadEditor v-model:editor-incomplete-upload="slotProps.data.editorObject.value" />
+          <div class="flex gap-3 justify-content-between">
+            <PVButton
+              icon="pi pi-trash"
+              class="p-button-danger p-button-outlined"
+              label="Delete"
+              @click="() => deleteIncompleteUpload(slotProps.data.id)"
+            />
+            <div v-tooltip.bottom="slotProps.data.saveTooltip">
+              <PVButton
+                :disabled="!slotProps.data.canSave.value"
+                label="Save Changes"
+                icon="pi pi-save"
+                icon-pos="right"
+                @click="() => saveChanges(slotProps.data.id)"
+              />
+            </div>
+          </div>
+          <StandardDebug
+            :value="slotProps.data.editorObject.value"
+            label="Incomplete Upload"
+          />
+        </div>
+      </template>
+    </PVDataTable>
+    <PVButton
+      v-if="editorObjects.length > 0"
+      label="Delete All Incomplete Uploads"
+      icon="pi pi-trash"
+      class="p-button-danger p-button-outlined"
+      @click="deleteAll"
+    />
+    <StandardDebug
+      :value="data"
+      label="Editor Objects"
+    />
+  </standardcontent>
+</template>
+
+<style lang="scss">
```

Comment:
nit: Any chance we can [use `scoped` here](https://vuejs.org/api/sfc-css-features.html)? It's great for avoiding CSS collisions as an app grows.